datachainlab/cosmos-ethereum-ibc-lcp
# bump geth and lodestar for e2e tests

- bump geth to v1.14.5
- bump lodestar to v1.19.0

## QA
```
$ make network                           
make -C ../../chains/tendermint network
make[1]: Entering directory '/home/ubuntu/git/datachainlab/cosmos-ethereum-ibc-lcp/tests/e2e/chains/tendermint'
/usr/bin/docker compose up -d
WARN[0000] The "LCP_RA_ROOT_CERT_HEX" variable is not set. Defaulting to a blank string. 
[+] Running 1/0
 ✔ Container tendermint-chain  Running                                                                                                                                                               0.0s 
make[1]: Leaving directory '/home/ubuntu/git/datachainlab/cosmos-ethereum-ibc-lcp/tests/e2e/chains/tendermint'
make -C ../../chains/ethereum network deploy extract-abi
make[1]: Entering directory '/home/ubuntu/git/datachainlab/cosmos-ethereum-ibc-lcp/tests/e2e/chains/ethereum'
GETH_VERSION=v1.14.5 GETH_HTTP_PORT=8546 \
LODESTAR_VERSION=v1.19.0 BEACON_HTTP_PORT=19596 \
docker compose up -d geth lodestar deposit lodestar-validator
WARN[0000] The "SGX_MODE" variable is not set. Defaulting to a blank string. 
[+] Running 4/4
 ✔ Container geth                Healthy                                                                                                                                                             0.5s 
 ✔ Container lodestar            Healthy                                                                                                                                                             1.0s 
 ✔ Container deposit             Exited                                                                                                                                                             26.3s 
 ✔ Container lodestar-validator  Running                                                                                                                                                             0.0s 
docker compose run contracts npx hardhat run ./scripts/deploy.js --network eth_local
WARN[0000] The "SGX_MODE" variable is not set. Defaulting to a blank string. 
Downloading compiler 0.8.20
Warning: SPDX license identifier not provided in source file. Before publishing, consider adding a comment containing "SPDX-License-Identifier: <SPDX-License>" to each source file. Use "SPDX-License-Identifier: UNLICENSED" for non-open-source code. Please see https://spdx.org for more information.
--> @ensdomains/ens-contracts/contracts/dnssec-oracle/BytesUtils.sol


Warning: SPDX license identifier not provided in source file. Before publishing, consider adding a comment containing "SPDX-License-Identifier: <SPDX-License>" to each source file. Use "SPDX-License-Identifier: UNLICENSED" for non-open-source code. Please see https://spdx.org for more information.
--> @ensdomains/ens-contracts/contracts/dnssec-oracle/algorithms/ModexpPrecompile.sol


Warning: SPDX license identifier not provided in source file. Before publishing, consider adding a comment containing "SPDX-License-Identifier: <SPDX-License>" to each source file. Use "SPDX-License-Identifier: UNLICENSED" for non-open-source code. Please see https://spdx.org for more information.
--> @ensdomains/ens-contracts/contracts/dnssec-oracle/algorithms/RSAVerify.sol


Warning: Unused function parameter. Remove or comment out the variable name to silence this warning.
   --> @hyperledger-labs/yui-ibc-solidity/contracts/proto/MockClient.sol:201:5:
    |
201 |     Data memory r
    |     ^^^^^^^^^^^^^


Warning: Unused function parameter. Remove or comment out the variable name to silence this warning.
  --> contracts/MockApp.sol:28:9:
   |
28 |         uint64 timeoutTimestamp
   |         ^^^^^^^^^^^^^^^^^^^^^^^


Warning: Unused function parameter. Remove or comment out the variable name to silence this warning.
  --> contracts/MockApp.sol:39:51:
   |
39 |     function onRecvPacket(Packet calldata packet, address relayer)
   |                                                   ^^^^^^^^^^^^^^^


Warning: Unused function parameter. Remove or comment out the variable name to silence this warning.
  --> contracts/MockApp.sol:49:94:
   |
49 |     function onAcknowledgementPacket(Packet calldata packet, bytes calldata acknowledgement, address relayer)
   |                                                                                              ^^^^^^^^^^^^^^^


Compiled 68 Solidity files successfully (evm target: paris).
RA simulation is disabled
Deploying the contracts with the account: 0xa89F47C6b463f74d87572b058427dA0A13ec5425
Account balance: 611999924664246584154
IBCHandler address: 0x3379866A055CF3725775e8d6876be4877e1a7D89
LCPProtoMarshaler address: 0xf68230F013AbBc6c2330cdE2Fa2dF84e2417B4a0
AVRValidator address: 0xa954B862442936E35B579EB475CE82768389D301
LCPClient address: 0x4efa02d32DDDf9856222331ABEFf1a09248FeB87
MockApp address: 0xC2D142Fda35551b0df9a52Cb0e5340B8Bec89166
npm notice
npm notice New minor version of npm available! 10.7.0 -> 10.8.1
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.8.1
npm notice To update run: npm install -g npm@10.8.1
npm notice
./scripts/extract-abi.sh
make[1]: Leaving directory '/home/ubuntu/git/datachainlab/cosmos-ethereum-ibc-lcp/tests/e2e/chains/ethereum'
